pkg/config/crd/store_test.go

```diff
@@ -33,6 +34,9 @@ import (
 	"istio.io/mixer/pkg/config/store"
 )
 
+// The "retryTimeout" used by the test.
+const testingRetryTimeout = 10 * time.Millisecond
```

Comment:
Can we make this as small as possible (1ms or less, ideally) - this kind of stuff adds up and makes tests take longer gradually over time.

Reply:
The value comes from my local environment, and it would be highly depending on the computing powers, amount of resources, goroutine scheduling, and so on. But I'm afraid that <1ms could be too short.

It's still hard (for me) to say what's the best value here 😢 added a TODO comment.

Reply:
Ah, wait -- I see another way, to use `HasSynced` property (I found `LastSyncResourceVersion` is racy, but `HasSynced` is not fortunately). Rewritten the code for the wait. PTAL.